### PR TITLE
Add class

### PR DESCRIPTION
**Pull request by saikatgpae**
This PR is created to achieve the same goal, using JS Class properties what we previously done using objects and arrays.
In this PR we are going to create objects using JS class properties and we will add or remove books using class method properties.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -22,18 +22,18 @@ <h1 class="desktop-heading">All awsome books</h1>
...
                         <div class="">
                             <button type="button" class="remove btn btn-outline-secondary mt-2" id="add">add</button>
                         </div>
                     </form>
         </section>
     </div>
-    <!-- <script src="./main.js"></script> -->
-    <script src="./index.js"></script>
+    <script src="./main.js"></script>
+    <!-- <script src="./index.js"></script> -->
```
Comment (vechicin) [inline index.html:37]: Please remove this line of commented code to keep a cleaner codebase.

File: main.js [hunk 2]
```diff
@@ -1,54 +1,83 @@
-// var list = [];
+/* eslint-disable */
```
Comment (vechicin) [inline main.js:1]: Please do not disable linters altogether. If you're having issues having multiple classes in the same file, just disable that rule but do not disable ESLint in its entirety.

File: main.js [hunk 3]
```diff
@@ -1,54 +1,83 @@
...
+var storedBooksList = JSON.parse(localStorage.getItem('books'));
+console.log(storedBooksList);
```
Comment (vechicin) [inline main.js]: Please remove this `console.log` to keep a cleaner codebase.

File: main.js [hunk 4]
```diff
@@ -1,54 +1,83 @@
...
-/* eslint-disable */
-for (let i = 0; i < storedBooksList.length; i++) {
-    /* eslint-enable */
-  console.log(storedBooksList[i].title, storedBooksList[i].author);
-  const book = document.createElement('ul');
-  book.className = 'book-menu';
-  book.innerHTML = `<li class="book-name list-group-item">${storedBooksList[i].title}</li>
+    add(title, author) {
+        if (Array.isArray(this.list)) {
+            this.list.push(new Book(title, author));
```
Comment (vechicin) [inline main.js]: [OPTIONAL] Consider using methods that don't mutate the array.

Screen Shot 2022-08-11 at 6 44 42

File: main.js [hunk 5]
```diff
@@ -1,54 +1,83 @@
...
+        }
+    }
+
+    removeItem(index) {
+        if (Array.isArray(this.list)) {
+            const array = this.list;
+            array.splice(index, 1);
```
Comment (vechicin) [inline main.js]: [OPTIONAL] Consider using methods that don't mutate the array.

Screen Shot 2022-08-11 at 6 45 22

File: main.js [hunk 6]
```diff
@@ -1,54 +1,83 @@
...
-  // storing the array inside the local storage
-  localStorage.setItem('books', JSON.stringify(list));
+// DELETE THE CURRENT ROW OF THE DATDABASE.
+function deleteRow() {
+    const index = this.previousElementSibling.innerHTML;
+    const books = new Books();
+    const booksStored = JSON.parse(localStorage.getItem('books'));
+    books.list = booksStored;
+    books.removeItem(index);
+    // console.log(books.list);
```
Comment (vechicin) [inline main.js]: Please remove this line of commented code to keep a cleaner codebase.

File: main.js [hunk 7]
```diff
@@ -1,54 +1,83 @@
...
-  document.getElementById('author').value = '';
-  console.log(storedBooksList.length);
-  // console.log(list[list.length-1].title);
+// ADD NEW BOOK.
+document.getElementById('add').addEventListener('click', () => {
+    const books = new Books();
+    const title = document.getElementById('title').value;
+    const author = document.getElementById('author').value;
+    books.add(title, author);
+    // console.log(books.list);
```
Comment (vechicin) [inline main.js]: Please remove this line of commented code to keep a cleaner codebase.

File: style.css [hunk 8]
```diff
@@ -27,7 +27,7 @@
     justify-content: space-between;
   }
 
-  .desktop-add-heading::before {
+  /* .desktop-add-heading::before {
```
Comment (vechicin) [inline style.css]: Please remove these lines of commented code to keep a cleaner codebase.

**Review by vechicin — CHANGES_REQUESTED** [review]
# Hi @saikatgpae and @OpondoG ✋, 

Good job so far! 🔥 
There are some issues that you still need to work on to go to the next project but you are almost there! 🚨 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights ⚡ 

- [x] Good job refactoring the project using classes!

## Required Changes ♻️

- [ ] There is a lot of commented code all throughout your project and, unfortunately, I couldn't provide inline comments for all of it. Please make sure you review all of your files and remove all of the commented code to keep a clean codebase.

_Check the comments under the review._

## Optional suggestions 📂 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (saikatgpae) [reply index.html:37]: removed the same.

File: main.js (hunk 2 above)
Comment (saikatgpae) [reply main.js:1]: Changed the same.

File: main.js (hunk 6 above)
Comment (saikatgpae) [reply main.js]: Removed the same.

File: main.js (hunk 7 above)
Comment (saikatgpae) [reply main.js]: Removed the same.

File: style.css (hunk 8 above)
Comment (saikatgpae) [reply style.css]: removed.

**Review by cynthiainga — APPROVED** [review]
Hi @saikatgpae, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- [ ] No linters errors ✔️ 
- [ ] Good commit history ✔️ 
- [ ] Descriptive PR ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**